app/alembic/versions/20251204181617_19b6f2ee95e6_add_parse_filters_to_projects.py

```diff
@@ -0,0 +1,34 @@
+"""add_parse_filters_to_projects
+
+Revision ID: 20251204181617_19b6f2ee95e6
+Revises: 20250928_simple_global_cache
+Create Date: 2025-12-04 18:16:17.895272
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision: str = '20251204181617_19b6f2ee95e6'
+down_revision: Union[str, None] = '20250928_simple_global_cache'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('projects', sa.Column('parse_filters', postgresql.JSONB(astext_type=sa.Text()), nullable=True))
```

Comment:
I thought that we had discussed that we would not add a new migration and use the existing projects table to store this information. Secondly I thought we already have a project updated at column. Why do we need a separate  last parsed at column?